src/types.ts

```diff
@@ -1,3 +1,15 @@
-import parsePokemon from "./hooks/utils/parsePokemon";
+import { PokemonTypes } from "./constants";
 
-export type Pokemon = ReturnType<typeof parsePokemon>;
+export type Pokemon = {
+  id: number;
+  name: string;
+  imageUri: string;
+  types: PokemonTypes[];
+  evolutions:
+    | {
+        __typename?: "pokemon_v2_pokemonspecies";
```

Comment:
Is this `typename` really required here? 🤔

Reply:
😱